### ISSUE TITLE
POST request not functioning as expected. 

### ISSUE THREAD

**Issue by jacobpascalcoblentz** [benoitdm-oslandia/pg_featureserv#166]
Creating a basic table as follows: 

```
CREATE TABLE public.jc_test (
fid INT PRIMARY KEY NOT NULL,
geom GEOMETRY(POINT, 4326)
);
```

Then running a `POST` request against it as:
```
curl --location --request POST 'http://localhost:9000/collections/public.jc_test/items' \
--header 'Content-Type: application/json' \
--data-raw '{ "type": "Feature", "geometry": { "type": "Point", "coordinates": [ -75.849253579389796, 47.6434349837781 ] } , "properties": { "fid": "1" }}'
```
returns this error: 
<img width="659" alt="Screen Shot 2023-02-10 at 12 00 59 PM" src="https://user-images.githubusercontent.com/75592265/218186547-e5011363-4102-4b30-983c-3d8b4699a8aa.png">

**Comment by benoitdm-oslandia** [comment]
Closed as duplicate of #160

**Comment by azarz** [comment]
Fixed in #161
